cli-plugins/apps/tutorials/delete-workload.hbs.md

```diff
@@ -0,0 +1,25 @@
+# Delete a workload
+
+Executing this command removes workloads from a cluster. However, it's important to note that 
```

Comment:
The delete command can be used to remove one or many workloads from a specific namespace on the target cluster. 
> **Note** this command _does not_ automatically remove a workload's corresponding image(s) from the registry.

Reply:
ok, edited